app/assets/javascripts/controllers/cloud_tenant/cloud_tenant_form_controller.js

```diff
@@ -1,14 +1,16 @@
 ManageIQ.angular.app.controller('cloudTenantFormController', ['$http', '$scope', 'cloudTenantFormId', 'miqService', function($http, $scope, cloudTenantFormId, miqService) {
-  $scope.cloudTenantModel = { name: '', ems_id: '' };
-  $scope.formId = cloudTenantFormId;
-  $scope.afterGet = false;
-  $scope.modelCopy = angular.copy( $scope.cloudTenantModel );
-  $scope.model = "cloudTenantModel";
+  var vm = this;
+
+  vm.cloudTenantModel = { name: '', ems_id: '' };
+  vm.formId = cloudTenantFormId;
+  vm.afterGet = false;
+  vm.modelCopy = angular.copy( vm.cloudTenantModel );
+  vm.model = "cloudTenantModel";
```

Comment:
Please add `vm.saveable = miqService.saveable;` to let button Add/Save know if form is dirty and valid.